dashboard/src/index.tsx

```diff
@@ -1,15 +1,16 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import * as Modal from "react-modal";
-import { createAxiosInterceptors } from "shared/AxiosInstance";
-import { axios } from "./shared/Auth";
+import * as Axios from "shared/AxiosInstance";
+import * as Auth from "./shared/Auth";
 
 import Root from "./containers/Root";
 import "./index.css";
 import store from "./store";
 // import registerServiceWorker from "./registerServiceWorker";
 
-createAxiosInterceptors(axios, store);
+Axios.createAxiosInterceptors(Axios.axios, store);
```

Comment:
we should add a comment explaining what we're doing here. It's also quite hard to read since `Axios.axios` and `Auth.axios` look very similar. Can we distinguish them better somehow?

Reply:
actually, I am not sure why are we doing this here? It works if I add the interceptors in `AxiosInstance`